robottelo/ui/subscription.py

```diff
@@ -1,13 +1,21 @@
 """Implements Subscriptions/Manifest handling for the UI"""
 
-from robottelo.helpers import escape_search
 from robottelo.ui.base import Base
 from robottelo.ui.locators import common_locators, locators
+from robottelo.ui.navigator import Navigator
 
 
 class Subscriptions(Base):
```

Comment:
It will never work as it will never find search_box. The only explanation that you run tests for another code which is definitely bad practice :)